__tests__/migrations.spec.js

```diff
@@ -223,6 +223,59 @@ describe('migrations', () => {
       })
     })
 
+    it('should ignore input that is already migrated', () => {
+      const newSchema = {
+        type: 'object',
+        properties: {
+          is_running_latest__release: {
+            title: 'Running Latest Release',
+            type: 'boolean',
+          },
+          device_tag: {
+            title: 'Tag',
+            type: 'object',
+            properties: {
+              tag_key: {
+                title: 'Name',
+                description: 'key',
+                type: 'string',
+              },
+              value: {
+                title: 'Value',
+                description: 'value',
+                type: 'string',
+              },
+            }
+          }
+        }
+      }
+
+      const oldFilter = {
+        type: 'Boolean',
+        name: 'is_running_latest__release',
+        label: 'Running Latest Release',
+        availableOperators: [
+          {
+            label: 'is true',
+            value: 'is true'
+          },
+          {
+            label: 'is false',
+            value: 'is false'
+          }
+        ],
+        operator: 'is true',
+        value: '',
+        id: 'tOYR2EqLICRfS1cS'
+      }
+
+      const migratedFilter = migrations.migrateLegacyFilter(newSchema, oldFilter)
+
+      expect(
```

Comment:
Could we also add an expectation for the new filter object as well and not just check that re-migrating is not altering the filter?

Reply:
Good idea